torch/fx/_symbolic_trace.py

```diff
@@ -800,9 +800,9 @@ def __deepcopy__(self, memo):
         return new_tracer
 
 
-# List of pairs of (global dict, function name) functions
+# List of triplets of (global dict, function name, visible_to_make_fx) functions
 # to patch for the purposes of the wrap() API.
-_wrapped_fns_to_patch: List[Tuple[dict, str]] = []
+_wrapped_fns_to_patch: List[Tuple[dict, str, bool]] = []
```

Comment:
`_wrapped_methods_to_patch.append((torch.Tensor, "__getitem__", False))` maybe? Please check if there are other assignments to this variable.

Reply:
I think we will leave _wrapped_methods_to_patch out of this. We will not be patching them, and there is no actual point in wrapping any method except on Tensor. See: https://github.com/pytorch/pytorch/issues/94470

Reply:
Humm, what if the user implementing their own tensor class without inheriting torch.Tensor? Probably I am dumb, but I feel this breaks the ability that Proxy can represent any type.

Reply:
> Proxy can represent any type

Actually, calling on methods on such a class is only possible if the class subclasses `metaclass=torch.fx.ProxyableClassMeta`. In which case, its methods would be wrapped by default.